syntax = "proto3";

package legumeinfo.microservices.chromosomesearch_service.v1;


service ChromosomeSearch {
  rpc Search (ChromosomeSearchRequest) returns (ChromosomeSearchReply) {}
}


message ChromosomeSearchRequest {
  string query = 1;
}


message ChromosomeSearchReply {
  repeated string chromosomes = 1;
}
